pages/common/nginx.md

```diff
@@ -8,4 +8,4 @@
 
 - Start server with custom config and custom relative path base directory
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`
```

Comment:
Right. But it is not {{path to config}} anyway. Maybe it is better to call it {{config_file_name}}.

Could you show me real world example?